### PR TITLE
Add stub document manager

### PR DESCRIPTION
**Pull request by m-doughty**
### Name of the story

N/A

### Acceptance Criteria

- [x] Stub API for Dilu's testing;
- [x] Documentation for Stub API;
- [x] Dockerize Stub API. 

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/document-manager/stub-api/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Send a DELETE or POST request for a document;
+- Continue to poll the GET endpoint for that document until the status has
+updated.
+
+In the background, the real API will asynchronously process the request and
+update the status of the document as it progresses.
+
+## Running
+
+**NOTE: THIS STILL NEEDS TO BE CONTAINERIZED.**
```
Comment (rjkerrison) [inline medicines/document-manager/stub-api/README.md]: looks like the Dockerfile runs `stub.pl`, so can the Docker run instructions be added now?

File: medicines/document-manager/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## About
+
+The Document Manager handles the batch import for creating & deleting
+new documents in Azure blob storage, which will be searchable within
+the Products website.
+
+## To-Do
+
+- Dockerize stub API;
```
Comment (rjkerrison) [inline medicines/document-manager/README.md]: can this one be marked as done?

Comment (m-doughty) [reply medicines/document-manager/README.md]: Yes.

File: medicines/document-manager/stub-api/README.md (hunk 1 above)
Comment (m-doughty) [reply medicines/document-manager/stub-api/README.md]: Yes.

**Pull request merged**